.travis.yml

```diff
@@ -4,56 +4,30 @@ sudo: false
 python:
   - "2.7"
   - "3.4"
+  - "3.5"
 
-# command to install dependencies, e.g. pip install -r requirements.txt --use-mirrors
+cache:
```

Comment:
Caches the miniconda and testdb.sqlite for next commit/test. Miniconda and all packages are still updated to the latest next time.
If you prefer to start from scratch when using testdb.sqlite we should correct that by adding a tearDownClass method that removes testdb and the pickle.